www/android/geolocation.js

```diff
@@ -52,7 +62,20 @@ module.exports = {
           var geo = cordova.require('cordova/modulemapper').getOriginalSymbol(window, 'navigator.geolocation');
           geo.clearWatch(args[0]);
         }
-        exec(win, error, "Geolocation", "getPermission", []);
+        var fail = function() {
+            if (error) {
+                error(illegalAccessError);
+            }
+        };
+        exec(win, fail, "Geolocation", "getPermission", []);
     }
 };
 
+// mimic PositionError to be compliant with W3C spec
+var illegalAccessError = {
```

Comment:
Insead of creating custom object, can we reuse [PositionError polyfill](https://github.com/apache/cordova-plugin-geolocation/blob/master/www/PositionError.js) here?